### PR TITLE
benchmark: sysbench 2.0 vs 1.0

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@shenli PTAL

**Comment by lilin90** [comment]
@CaitinChen PTAL and align in docs. Thanks!

File: benchmark/sysbench-v2.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+### 标准 OLTP 测试 
+
+| 版本 | table count | table size | sysbench threads | tps | qps | latency(avg / .95) | 
+| :---: | :---: | :---: | :---: | :---: | :---: | :---:|
+| v2.0.0-rc6 | 32 | 1000 万 | 128 * 3 | 5404.22 | 108084.4 | 87.2033 ms / 110 ms |
+| v2.0.0-rc6 | 32 | 1000 万 | 256 * 3 | 5578.165 | 111563.3 | 167.673 ms / 275.623 ms |
+| v2.0.0-rc6 | 32 | 1000 万 | 512 * 3 | 5874.045 | 117480.9 | 315.083 ms / 674.017 ms |
+| v2.0.0-rc6 | 32 | 1000 万 | 1024 * 3 | 6290.7 | 125814 | 529.183 ms / 857.007 ms |
+| v2.0.0-rc6 | 32 | 1000 万 | 2048 * 3 | 6212.55 | 124251 | 981.143 ms / 1328.05 ms |
```
Comment (shenli) [inline benchmark/sysbench-v2.md]: 这一个删掉吧，并发有点大

**Comment by shenli** [comment]
LGTM

**Comment by zhengwanbo** [comment]
LGTM

**Pull request merged**